search-app/resources/templates/sitemap-provider-tag.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+ {% for link in links %}
+  <url>
+    <loc>{{ link.href }}</loc>
+    <lastmod>{{ datestamp }}</lastmod>
```

Comment:
It seems like we should include the <changefreq> tag as well. Are we regenerating the sitemap pages daily?